src/main/scala/sbtorgpolicies/rules/rules.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtorgpolicies
+
+import cats.syntax.either._
+import jdk.nashorn.internal.runtime.regexp.RegExp
+import sbtorgpolicies.exceptions.ValidationException
+
+import scala.util.matching.Regex
+
+package object rules {
+
+  type ValidationResult = Either[ValidationException, Unit]
```

Comment:
If instead of using `Either` as a data type to signal errors you use `ValidatedNel[E, A]` we can do a parallel pass and validate it all at once.